aws/aws_test.go

```diff
@@ -138,9 +138,6 @@ func Test_Examples(t *testing.T) {
 				testURLGet(t, nginxEndpoint, "", "<h1> Hi from Pulumi </h1>")
 			},
 		},
-	}
-
-	longTests := []integration.ProgramTestOptions{
 		{
```

Comment:
this is temporary just to validate that tests pass.  once they do i'll put htis back in the long-tests group.